compiler/plugins/input/StableHLO/PluginRegistration.cpp

```diff
@@ -107,6 +110,10 @@ struct StableHLOSession
       buildStableHLOXLAInputConversionPassPipeline(passManager,
                                                    stableHloOptions);
       return true;
+    } else if (typeMnemonic == "vhlo") {
+      ::mlir::stablehlo::createStablehloDeserializePipeline(passManager);
+      buildStableHLOInputConversionPassPipeline(passManager, stableHloOptions);
```

Comment:
Could we add this `createStablehloDeserializePipeline` into the StableHLO input conversion pipeline?